mono/metadata/object-internals.h

```diff
@@ -1709,8 +1712,11 @@ mono_class_compute_gc_descriptor (MonoClass *klass);
 gsize*
 mono_class_compute_bitmap (MonoClass *klass, gsize *bitmap, int size, int offset, int *max_set, gboolean static_fields);
 
-MonoObject*
-mono_object_xdomain_representation (MonoObject *obj, MonoDomain *target_domain, MonoError *error);
+MonoObject* // FIXMEcoop remove this
```

Comment:
Line damage; we know any `MonoObject*` needs to be removed, commenting doesn't bring any more explanation to why it needs to be done.

Reply:
Sure.